pkg/cli/admin/node/logs.go

```diff
@@ -173,6 +190,18 @@ func (o LogsOptions) Validate() error {
 	if o.BootChanaged && (o.Boot < -100 || o.Boot > 0) {
 		return fmt.Errorf("--boot accepts values [-100, 0]")
 	}
+	if len(o.Query) > 0 && (len(o.Path) > 0 || len(o.Units) > 0) {
+		return fmt.Errorf("--path and --unit flags are mutually exclusive with --query")
+	}
+	if len(o.SinceTime) > 0 && (len(o.Since) > 0 || len(o.Until) > 0) {
+		return fmt.Errorf("cannot combine --since-time with --since or --until")
```

Comment:
Can we also change this in the format of https://github.com/openshift/oc/pull/1403/files#diff-fe5df7f76dde8b46f44cbcfa239451ec304c914cfc3d6a17087debb317079ee0R194?

Reply:
Done